docs/programmatic-api.md

```diff
@@ -255,6 +255,9 @@ await testplane.readTests(testPaths, options);
     * **ignore** (optional) `String|Glob|Array<String|Glob>` - patterns to exclude paths from the test search.
     * **sets** (optional) `String[]`– Sets to run tests in.
     * **grep** (optional) `RegExp` – Pattern that defines which tests to run.
+    * **replMode** (optional) `{enabled: boolean; beforeTest: boolean; onFail: boolean;}` - [Test development mode using REPL](./cli.md#--repl). When reading the tests, it checks that only one test is running in one browser.
+    * **runnableOpts** (optional):
```

Comment:
new option which gives me ability to save test locations in file (`line` and `column`)